src/software/ai/hl/stp/tactic/tactic_visitor.h

```diff
@@ -52,10 +52,10 @@ class TacticVisitor
     virtual void visit(const BlockShotPathTactic &tactic) = 0;
     virtual void visit(const MoveTactic &tactic) = 0;
     virtual void visit(const ChipTactic &tactic) = 0;
-    virtual void visit(const KickoffChipTactic &tactic) = 0;
+    virtual void visit(const KickoffChipTactic &tactic)        = 0;
```

Comment:
nit: remove extra indents (?)

Reply:
this was caused by fix formatting

Reply:
Huh, was fix formatting applied on the diff (is that even a thing?) or on all files? This inconsistency is... unnerving

Reply:
Diff against upstream/master. Yeah not sure how this passed CI

Reply:
Can you run `fix_formatting.sh -a` and see what happens?

Reply:
Looks like a decent chunk of files got changed with `-a`. This makes sense as we generally just check formatting on the diff.  

The historical context here is that we originally checked only on the diff because we didn't want to have to format the entire repository at once back when we introduced clang-format.

Might be good to have a quick discussion on how to address this, but definitely out-of-scope for this PR.

Reply:
I think what's happening is that CI doesn't check firmware for formatting.
Anything wrong with pushing the formatting changes in this PR to master though?

Reply:
Pretty sure CI does check firmware for formatting since it just checks the output of the `fix_formatting.sh` script, which is also what we use to format the code.